docs/data_fields.md

```diff
@@ -0,0 +1,28 @@
+
+The main fields are the following, plus the geographical entity fields explained below:
+
+* date: the date the data point refers to
+* type: the type of data point: cases, tested, deaths, hospitalized, discharged, or recovered
+* value: the value of the data point (a count of events of a certain type)
+
+Each observation is associated with a geographical entity, identified in the following four fields.
+
+
+* city: the name of the city
+* county: county or parish or the appropriate name of the administrative subdivision below the level of state or equivalent 
+* state: state, province or region depending on jurisdiction. In general, the first administrative subdivision below the level of country.
+* country:  ISO 3166-1 alpha-3 (three letter) country code 
+
+The country field is mandatory. In general, whenever the record is about administrative subdivisions of a level, al including levels are non empty. One exception is New York City because it is subdivided into five counties.
+
+The following fields are uniquely determined by the geographical entity and are provided as a convenience.
+
+* population: a recent estimate of the population in the geographical entity
```

Comment:
Do we know more about the source of the pop estimate

Reply:
Yes, it will come from https://github.com/hyperknot/country-levels and other census sources we have in the repo.

Reply:
I was thinking more about original source, like is it world bank, UN, US census etc, what year and so on. The exact file we keep it in is of little consequence for a data consumer.